src/event/event_config.h

```diff
@@ -133,7 +133,7 @@
 #	define EV_ONESHOT				0x0010
 #	define EV_CLEAR					0x0020
 #	define EV_DISPATCH				0x0080
-#	define EV_UDATA_SPECIFIC		0x0100
+#	define EV_UDATA_SPECIFIC			0x0000
```

Comment:
My recollection is that @MadCoder preferred us to define it as 0x0000 instead of leaving it undefined.  I believe it is a matter of style; I think either leaving it undefined or defining it to be 0x0000 accomplish the same thing.